inc/Taxonomies_Core.php

```diff
@@ -119,10 +119,9 @@ public function render_taxonomy_filters( $post_type ) {
 			}
 
 			$value = isset( $_REQUEST[ $taxonomy->query_var ] ) ? $_REQUEST[ $taxonomy->query_var ] : '';
-
 			?>
-			<label for="attachment-<?php echo sanitize_html_class( $taxonomy->slug ); ?>-filter" class="screen-reader-text"><?php echo esc_html( $this->get_filter_by_label( $taxonomy ) ); ?></label>
-			<select class="attachment-filters" name="<?php echo esc_attr( $taxonomy->query_var ); ?>" id="attachment-<?php echo sanitize_html_class( $taxonomy->slug ); ?>-filter">
+			<label for="attachment-<?php echo sanitize_html_class( $taxonomy->query_var ); ?>-filter" class="screen-reader-t ext"><?php echo esc_html( $this->get_filter_by_label( $taxonomy ) ); ?></label>
```

Comment:
"screen-reader-t ext" should be "screen-reader-text" once that's fixed you can merge it in!